tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/util/ReadConfig.java

```diff
@@ -0,0 +1,31 @@
+package ca.bc.gov.open.jagefilingapi.qa.util;
+
+import org.apache.log4j.Logger;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ReadConfig {
+
+    Properties prop;
+
+    public ReadConfig() throws IOException {
+            File src = new File("src\\main\\java\\ca\\bc\\gov\\open\\jagefilingapi\\qa\\config\\config.properties");
```

Comment:
is there no way to pass in a relative path?
something like
```
../../config/config.properties
```

Reply:
I tried using the suggested relative path prior to using the existing one. Couldn't get to work. Assumption was due to the package name in the path. Will keep trying solutions. Thanks.